scripts/publish-sentry-release.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -o errexit -o pipefail
+
+# This script packages our JS source files and sourcemaps as a Sentry release.
+# https://docs.sentry.io/product/releases/
+
+source ./scripts/common.sh
+
+build_dir="public"
```

Comment:
NIT: As a good practice, you probably want to declare these with `readonly` like, `readonly build_dir="public"`.